containers/profile/timeline.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <Timeline :class="{'collapsed': collapsed}" load class="padded">
```

Comment:
The ES6 object shorthand looks a lot better in cases like this
```suggestion
  <Timeline :class="{collapsed}" load class="padded">
```

Reply:
Hell, that's great!